### PR TITLE
Enhancement: Add dotfiles for @localheinz

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] adds dotfiles for @localheinz

### CODE REVIEW

**Comment by localheinz** [comment]
@schlueter 

Wondering, is it also possible to install https://github.com/ndbroadbent/scm_breeze, for example?

File: amoller.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+---
+- hosts: all
+  vars:
+    user: amoller
+    dotfiles_user: amoller
+    dotfiles:
+      - gitconfig
+      - zshrc
+  roles:
+    - user-dotfiles
```
Comment (schlueter) [inline amoller.yml]: do you wanna add the *__oh-my-zsh__* role?

Comment (localheinz) [inline amoller.yml]: @schlueter 

Does that give me `oh-my-zsh`?

If so, totally! Give me moment!

**Comment by schlueter** [comment]
:+1:

**Comment by localheinz** [comment]
Thank you, @schlueter!

**Pull request merged**